gramps/gui/views/listview.py

```diff
@@ -225,7 +225,24 @@ def define_actions(self):
                          tip=self.EDIT_MSG,
                          callback=self.edit)
 
-    def build_columns(self):
+    def build_columns(self, preserve_col=True):
+        """
+        build the columns
+        """
+        # Preserve the column widths if rebuilding the view.
+        if self.columns and preserve_col:
```

Comment:
This duplicates code in the `on_delete` method.

Reply:
It does, in part.  Are you suggesting a change?